### ISSUE TITLE
`slc loopback:datasource` with `other` gives wrong instructions

### ISSUE THREAD

**Issue by superkhau** [strongloop/generator-loopback#231]
When scaffolding new apps, `slc` asks:

```
? Enter the connector name without the loopback-connector- prefix:
```

It only work properly when you add the prefix not without (ie. you have to enter in `loopback-connector-redis` instead of just `redis`, same for the kv connectors).

I propose we change the message or fix the bug (either TOB or at least change the message for the time being if we don't find time to work on it soon).

@bajtos @0candy @jannyHou

**Comment by bajtos** [comment]
+1, moved to `#tob` with `P1`

**Comment by jannyHou** [comment]
I suggest rewording the msg to ask for **FULL** name. 

Some components also need full name, e.g.: loopback-component-storage

**Comment by superkhau** [comment]
or module name/package.json/npmjs name instead of "full" which might be ambiguous.

**Comment by jannyHou** [comment]
Closing since https://github.com/strongloop/generator-loopback/pull/242 checked in.
